mlflow/utils/process.py

```diff
@@ -75,6 +77,18 @@ def _exec_cmd(
 
     env = env if extra_env is None else {**os.environ, **extra_env}
 
+    if is_in_databricks_runtime():
+        if python_path_env := os.environ.get("PYTHONPATH"):
+            python_paths = python_path_env.split(":")
+            filtered_paths = []
+            for p in python_paths:
+                try:
+                    os.stat(p)
+                    filtered_paths.append(p)
+                except (PermissionError, FileNotFoundError):
+                    pass
+            os.environ["PYTHONPATH"] = ":".join(filtered_paths)
```

Comment:
Can we do this in the spark UDF?

Reply:
I prefer to do it for all sub-processes, because invalid path inside PYTHONPATH might affect all python workloads.